paien.yaml

```
@@ -256,6 +256,7 @@ packages:
       - bedtools2@2.27.1
       - intel-tbb@2018.2
       - eigen@3.3.4~fftw~suitesparse~metis~scotch~mpfr
+      - qt@5.10.0 +opengl+webkit ^mesa 
```

Do we need the `^mesa`, it seems to concretize just fine without it.

Also, as (at least) `glib` and `mesa` depend on Python we need to add an explicit dependency to one of our sanctioned Python installations, maybe:
```suggestion
      - qt@5.10.0 +opengl+webkit ^python@3.6.5+optimizations+tk
```